cmd/harmony/main.go

```diff
@@ -229,21 +230,56 @@ func setupInitialAccount() (isLeader bool) {
 	return isLeader
 }
 
-func setupConsensusKey(nodeConfig *nodeconfig.ConfigType) *bls.PublicKey {
-	consensusPriKey, err := blsgen.LoadBlsKeyWithPassPhrase(*blsKeyFile, blsPassphrase)
+func readMultiBlsKeys(nodeConfig *nodeconfig.ConfigType) error {
+	multiKeyFolder := "./hmy/multikeys"
+	fileList, err := ioutil.ReadDir(multiKeyFolder)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "ERROR when loading bls key, err :%v\n", err)
-		os.Exit(100)
+		return err
 	}
-	pubKey := consensusPriKey.GetPublicKey()
 
-	// Consensus keys are the BLS12-381 keys used to sign consensus messages
-	nodeConfig.ConsensusPriKey, nodeConfig.ConsensusPubKey = consensusPriKey, consensusPriKey.GetPublicKey()
-	if nodeConfig.ConsensusPriKey == nil || nodeConfig.ConsensusPubKey == nil {
-		fmt.Println("error to get consensus keys.")
-		os.Exit(100)
+	for _, fileInfo := range fileList {
+		if strings.HasSuffix(fileInfo.Name(), ".key") {
```

Comment:
add some instructions on the format of the multikey folder.

Reply:
I am planning to update doc for that